plugins/modules/system/homectl.py

```diff
@@ -0,0 +1,618 @@
+            - This can also be used to add a comment to maintain compatability from C(useradd).
+        aliases: [ 'comment' ]
+        type: str
+    realm:
+        description:
+            - The 'realm' a user is defined in.
+        type: str
+    email:
+        description:
+            - The email address of the user.
+        type: str
+    location:
+        description:
+            -  A free-form location string describing the location of the user.
+        type: str
+    iconname:
+        description:
+            - The name of an icon picked by the user, for example for the purpose of an avatar.
+            - Should follow the semantics defined in the Icon Naming Specification.
+            - See U(https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html) for specifics.
+        type: str
+    homedir:
+        description:
+            - Path to use as home directory for the user.
+            - This is the directory the user's home directory is mounted to while the user is logged in.
+            - This is not where the user's data is actually stored, see I(imagepath) for that.
+            - Only used when a user is first created.
+        type: path
+    imagepath:
+        description:
+            - Path to place the user's home directory.
+            - See U(https://www.freedesktop.org/software/systemd/man/homectl.html#--image-path=PATH) for more information.
+            - Only used when a user is first created.
+        type: path
+    uid:
+        description:
+            - Sets the UID of the user.
+            - If using I(gid) homed requires the value to be the same.
+            - Only used when a user is first created.
+        type: int
+    gid:
+        description:
+            - Sets the gid of the user.
+            - If using I(uid) homed requires the value to be the same.
+            - Only used when a user is first created.
+        type: int
+    mountopts:
+        description:
+            - String separated by comma each indicating mount options for a users home directory.
+            - Valid options are C(nosuid), C(nodev) or C(noexec).
+            - Homed by default uses C(nodev) and C(nosuid) while C(noexec) is off.
+        type: str
+    umask:
+        description:
+            - Sets the umask for the user's login sessions
+            - Value from C(0000) to C(0777).
+        type: int
+    memberof:
+        description:
+            - String separated by comma each indicating a UNIX group this user shall be a member of.
+            - Groups the user should be a member of should be supplied as comma separated list.
+        aliases: [ 'groups' ]
+        type: str
+    skeleton:
+        description:
+            - The absolute path to the skeleton directory to populate a new home directory from.
+            - This is only used when a home directory is first created.
+            - If not specified homed by default uses ``/etc/skel``
```

Comment:
```suggestion
            - If not specified homed by default uses C(/etc/skel).
```
Only ever use double backticks in documentation if `C(...)` cannot be used (because the text to quote contains `(` or `)`). Same below.

Reply:
Thanks, sorry I am still getting used to the C() I() etc..

Reply:
This still needs to be changed :)